### DISCUSSION TITLE
IBAX PROJECT

### DISCUSSION THREAD

**Discussion by Janaka9310** [IBAX-io/go-ibax#2089] [Ideas]
IBAX PROJECT IS GOOD PROJECT AND DEV IS BEST

**Comment by Dikdik0109** [comment]
Nice

**Reply by REZWANA1** [reply]
Nice

**Comment by Ariful369** [comment]
Wow

**Comment by ithot01** [comment]
Ibax I believe this is a faithful project.The projector has a lot of attractions so hopefully the project will be better in the future and will

**Comment by zeland2323** [comment]
Thanks
